src/components/Photo.jsx

```diff
@@ -1,15 +1,19 @@
 import styles from '../styles/photo'
 
 import React from 'react'
+import { Link } from 'react-router'
 
-const STACK_FILES_DOWNLOAD_PATH = 'http://cozy.local:8080/files/download'
+import { STACK_FILES_DOWNLOAD_PATH } from '../constants/config'
 
-const Photo = ({ photo }) => {
+export const Photo = ({ photo, router }) => {
```

Comment:
Actually, the router is passed through the withRouter() HOC and the parent component doesn't use the router here so it couldn't pass the pathname only here.

And for the key attribute, it's for the 'React' way, since the tests said me the key is missing, even it's not an error, I added it. But preact itself seems to not need it. What's your opinion? /poke @goldoraf